plasmapy/simulation/particle_tracker.py

```diff
@@ -351,7 +373,7 @@ def __init__(
         self,
         grids: Union[AbstractGrid, Iterable[AbstractGrid]],
         termination_condition: AbstractTerminationCondition = None,
-        save_routine: Optional[AbstractSaveRoutine] = None,
+        save_routine: AbstractSaveRoutine = None,
```

Comment:
You could also just set the default here, instead of in the if statement below? 

`save_routine: Optional[AbstractSaveRoutine] = DoNotSaveSaveRoutine()`